oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -42,12 +43,17 @@ def __init__(self, server_hardware, server_hardware_types, server_profile):
                 server_hardware: ServerHardware dict from OneView
                 server_hardware_types: ServerHardwareTypes dict from OneView
                 server_profile: ServerProfile dict from OneView.
+                drives: Drives list from OneView
         """
         super().__init__(self.SCHEMA_NAME)
 
         base_resource = server_profile
         self.server_hardware = server_hardware
-
+        self.resource_block_uuids = list()
+        self.resource_block_uuids.append(self.server_hardware["uuid"])
+        self._fill_spt_on_resource_blocks_uuids_list(
```

Comment:
let's call "_fill_network_on..." because it represents the network resource block.